src/features/loading.ts

```diff
@@ -0,0 +1,30 @@
+import { configureStore } from "@reduxjs/toolkit";
+
+export const LOADING = "LOADING";
+export type LoadingAction = {
+  type: typeof LOADING;
+  load: boolean;
+};
+export const reload: (load: boolean) => LoadingAction = (load) => ({
```

Comment:
`reload: (load: boolean)` - this parameter name seems to be a bit confusing, if using boolean parameters, it is good to have them phrased in some sort of predicate, something like `isLoaded`, `isVisible` etc, `load` does not say that much.